packages/common/src/groups/HubGroups.ts

```diff
@@ -119,8 +119,20 @@ export async function createHubGroup(
     group,
     authentication: requestOptions.authentication,
   };
-  const result = await createGroup(opts);
-  return convertGroupToHubGroup(result.group, requestOptions);
+  const createdGroup = await createGroup(opts).then(async (res) => {
```

Comment:
seems slightly odd to use `.then` chaining here when it'a an async function already. Consider usign `await`